CalculatorApp/Program.cs

```diff
@@ -11,14 +11,14 @@
     if (input == null)
         break;
     input = input.Trim();
-    if (input.ToLower() == "q")
+    if (input.Equals("q", StringComparison.OrdinalIgnoreCase) || input.Equals("quit", StringComparison.OrdinalIgnoreCase))
         break;
-    if (input.ToLower() == "help")
+    if (input.Equals("h", StringComparison.OrdinalIgnoreCase) || input.Equals("help", StringComparison.OrdinalIgnoreCase))
     {
         Console.WriteLine("Calculator Help:");
         Console.WriteLine("- Enter mathematical expressions using +, -, *, /, ^");
```

Comment:
Expand on the supported functionality -- notably parentheses and negation
```suggestion
        Console.WriteLine("- Enter mathematical expressions using +, -, *, /, ^");
        Console.WriteLine("- Parentheses and negative numbers are supported.");
````